diffpy/snmf/factorizers.py

```diff
@@ -2,33 +2,34 @@
 import scipy.optimize
 
 
-def lsqnonneg(coefficient, target):
+def lsqnonneg(stretched_component_matrix, target_signal):
     """
-    Solves ``argmin_x || Ax - b ||_2`` for ``x>=0``.
+    Solves ``argmin_x || Ax - b ||_2`` for ``x>=0``. Finds the weights of component signals given unfactorized signal
```

Comment:
I would do it like this:
```
"""Finds the weights of component signals given unfactorized signal data and stretched components

Solves ``argmin_x || Ax - b ||_2`` for ``x>=0`` where A is the stretched_component_matrix and b is the target

Parameters
------------
...
"""
```
Please quickly read the pep8 and numpy standards for writing docstrings (for example, first line <80 chars followed by blank line etc....  Also google around for articles about making good docstrings.

If you learn this now it will become second nature and save a lot of time moving forward.

Reply:
I will read through the documentation and update the docstring.